optuna/storages/_rdb/alembic/versions/v2.0.0.a.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: v2.0.0.a
+Revises: v1.3.0.a
+Create Date: 2020-06-22 23:08:53.804043
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "v2.0.0.a"
+down_revision = "v1.3.0.a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("trials", sa.Column("datetime_last_update", sa.DateTime(), nullable=True))
```

Comment:
How about setting `datetime.now` as a default value instead of `nullable=True`?

Reply:
Thanks, that sounds reasonable. I used the server timestamp so that it's aligned to the on-UPDATE logic. PTAL.

Reply:
SQLite's `ALTER TABLE` doesn't seem to allow `CURRENT_TIMESTAMP` as a default value, so we'll have to modify `upgrade`. I'm not sure we can do that while keeping the same DDL.

```
E       sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) Cannot add a column with non-constant default
E       [SQL: ALTER TABLE trials ADD COLUMN datetime_last_update DATETIME DEFAULT (CURRENT_TIMESTAMP) NOT NULL]
E       (Background on this error at: http://sqlalche.me/e/e3q8)
```

Reply:
Addressed with batch updates. https://alembic.sqlalchemy.org/en/latest/batch.html